src/pudl/package_data/eia860/column_maps/energy_storage_existing.csv

```diff
@@ -0,0 +1,38 @@
+prime_mover_code,prime_mover,prime_mover,prime_mover,prime_mover,prime_mover,prime_mover,prime_mover
+sector_name_eia,sector_name,sector_name,sector_name,sector_name,sector_name,sector_name,sector_name
+sector_id_eia,sector,sector,sector,sector,sector,sector,sector
+capacity_mw,nameplate_capacity_mw,nameplate_capacity_mw,nameplate_capacity_mw,nameplate_capacity_mw,nameplate_capacity_mw,nameplate_capacity_mw,nameplate_capacity_mw
+summer_capacity_mw,summer_capacity_mw,summer_capacity_mw,summer_capacity_mw,summer_capacity_mw,summer_capacity_mw,summer_capacity_mw,summer_capacity_mw
+winter_capacity_mw,winter_capacity_mw,winter_capacity_mw,winter_capacity_mw,winter_capacity_mw,winter_capacity_mw,winter_capacity_mw,winter_capacity_mw
+storage_operating_month,operating_month,operating_month,operating_month,operating_month,operating_month,operating_month,operating_month
+storage_operating_year,operating_year,operating_year,operating_year,operating_year,operating_year,operating_year,operating_year
+energy_storage_capacity_mwh,nameplate_energy_capacity_mwh,nameplate_energy_capacity_mwh,nameplate_energy_capacity_mwh,nameplate_energy_capacity_mwh,nameplate_energy_capacity_mwh,nameplate_energy_capacity_mwh,nameplate_energy_capacity_mwh
+maximum_charge_rate_mw,maximum_charge_rate_mw,maximum_charge_rate_mw,maximum_charge_rate_mw,maximum_charge_rate_mw,maximum_charge_rate_mw,maximum_charge_rate_mw,maximum_charge_rate_mw
+maximum_discharge_rate_mw,maximum_discharge_rate_mw,maximum_discharge_rate_mw,maximum_discharge_rate_mw,maximum_discharge_rate_mw,maximum_discharge_rate_mw,maximum_discharge_rate_mw,maximum_discharge_rate_mw
+storage_electro_chemical_technology_type_1,storage_technology_1,storage_technology_1,storage_technology_1,storage_technology_1,storage_technology_1,storage_technology_1,storage_technology_1
+storage_electro_chemical_technology_type_2,storage_technology_2,storage_technology_2,storage_technology_2,storage_technology_2,storage_technology_2,storage_technology_2,storage_technology_2
+storage_electro_chemical_technology_type_3,storage_technology_3,storage_technology_3,storage_technology_3,storage_technology_3,storage_technology_3,storage_technology_3,storage_technology_3
+storage_electro_chemical_technology_type_4,storage_technology_4,storage_technology_4,storage_technology_4,storage_technology_4,storage_technology_4,storage_technology_4,storage_technology_4
+reactive_power_rating,nameplate_reactive_power_rating,nameplate_reactive_power_rating,nameplate_reactive_power_rating,nameplate_reactive_power_rating,nameplate_reactive_power_rating,nameplate_reactive_power_rating,nameplate_reactive_power_rating
+storage_enclosure_type,storage_enclosure_type,storage_enclosure_type,storage_enclosure_type,storage_enclosure_type,storage_enclosure_type,storage_enclosure_type,storage_enclosure_type
+serves_arbitrage_applications,arbitrage,arbitrage,arbitrage,arbitrage,arbitrage,arbitrage,arbitrage
```

Comment:
huh i actually think it would be more clear and true to the EIA documentation to use `served_`

> Indicates if this energy storage device **served** arbitrage applications **during the reporting year**

When you brought this up on slack w/ `serves_` i was assuming that it was a static or semi-static attribute of the storage generator, but EIA's docs seem to indicate that each of these are functions that the generator performed or did not perform so i think past tense is more clear and accurate.

Reply:
ok! that's fine with me